tests/Unit/Models/GroupTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Tests\Unit\Models;
+
+use App\Models\Bookmark;
+use App\Models\Group;
+use App\Models\Group\Member;
+use App\Models\Group\Workspace;
+use App\Models\LegiScan\Bill;
+use App\Models\User;
+use Illuminate\Support\Facades\DB;
+use Tests\TestCase;
+
+class GroupTest extends TestCase
+{
+    /** @test */
+    public function groupsHaveMembers()
+    {
+        $group = Group::factory()->create();
+        $user1 = User::factory()->create();
+        $user2 = User::factory()->create();
+        User::factory()->count(4)->create();
+        Member::factory()->create([
+            'group_id' => $group->id,
+            'user_id'  => $user1->id,
+        ]);
+        Member::factory()->create([
+            'group_id' => $group->id,
+            'user_id'  => $user2->id,
+        ]);
+
+        $this->assertEquals(2, $group->members->count());
+        $this->assertEquals(
+            [$user1->id, $user2->id],
+            $group->members->pluck('id')->sort()->toArray(),
+        );
```

Comment:
First assertion is 
```
$this->assertInstanceOf(
  GroupMember::class,
  $group->members->first()
);
```

Always assert that the thing you're getting back is of the type you expect.

Then just do the second assertion you have here. Logically you have proved anything more by asserting that the count is 2. It's basically redundant, and the second assertion has higher fidelity.